packages/ui-component-examples/lib/component-examples-loader.js

```diff
@@ -96,21 +92,10 @@ config.maxExamples = Boolean(config.maxExamples) ? config.maxExamples : ${
         config.maxExamples
       }
 
-if (!config.renderPage) {
-  console.warn('A default is no longer provided for \`renderPage\`, you should supply it to the config instead')
-}
-
-if (!config.renderExample) {
-  console.warn('A default is no longer provided for \`renderExample\`, you should supply it to the config instead')
-}
```

Comment:
here they likely wanted to check the opposite (`if (config.renderExample)`), see https://github.com/instructure/instructure-ui/commit/3abd6cac709de54e1be2eee3b5a6c22c0bdadb7b#diff-f82b6242f20fe2b1c9d6cf5c3522ad1c9463b3a9d8bd703c307050df0f079007